packages/common/src/SVGWidget.ts

```diff
@@ -410,6 +410,68 @@ export class SVGWidget extends Widget {
         return Math.sqrt((pointA.x - pointB.x) * (pointA.x - pointB.x) + (pointA.y - pointB.y) * (pointA.y - pointB.y));
     }
 
+    //  Download  ---
+    private serializeSVG(svg?: Element): Blob {
+        svg = svg || this.locateSVGNode(this._element.node());
+        (Array as any).from(svg.querySelectorAll("*")).forEach(elm => {
+            const styles = window.getComputedStyle(elm);
+            elm.style.font = styles.getPropertyValue("font");
+            elm.style.fill = styles.getPropertyValue("fill");
+            elm.style.stroke = styles.getPropertyValue("stroke");
+            elm.style.strokeWidth = styles.getPropertyValue("stroke-width");
+            elm.style.strokeDasharray = styles.getPropertyValue("stroke-dasharray");
+            elm.style.shapeRendering = styles.getPropertyValue("shape-rendering");
+            elm.style.opacity = styles.getPropertyValue("opacity");
+        });
```

Comment:
Won't this change the styles in the current web page - which potentially will break the page (I know your only "burning in" the existing styles)?